### ISSUE TITLE
environment.yml

### ISSUE THREAD

**Issue by gfelot** [godatadriven-dockerhub/airflow#4]
How do you use a mounted volume pointed to environment.yml to install package ?

I know that the image puckel/docker-airflow have a script that install a requirements.txt file.

**Comment by NielsZeilemaker** [comment]
Hi @gfelot, currently we don't support that, but it's a nice feature request. I'll have a look to see if I can implement something like that.

**Comment by gfelot** [comment]
Thank @NielsZeilemaker for your fast response. So how do you advise to use this image with miniconda and to install everything you need to work with ?

Here is how it's implemented in the entrypoint.sh of puckel image : https://github.com/puckel/docker-airflow/blob/master/script/entrypoint.sh#L34

I'm not skilled enough to help you with a PR on this.

My final goal is to have something ready with minconda, airflow, jupyter NB in a docker-compose.

**Comment by NielsZeilemaker** [comment]
I'll try to whip something together tomorrow. Should be pretty similar to solution of puckel.
Right now I would extend the current container with your own dockerfile, eg something like:
```
FROM godatadriven/airflow:1.10
ADD environment.yml /environment.yml
RUN conda install -f /environment.yml
```
It's bound to be a bit more complex than that, because you need to make sure it installs in the root venv, not create a new one.

**Comment by NielsZeilemaker** [comment]
I've merged #5 which allows you to add the dependencies. Let me know if it works for you.
